server/pom.xml

```diff
@@ -1,248 +1,219 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.aol.one.reporting</groupId>
-  <artifactId>forecast-api</artifactId>
-  <packaging>war</packaging>
-  <version>${ifs.version}</version>
-  <name>Forecast API Maven Webapp</name>
-  <url>http://maven.apache.org</url>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.aol.one.reporting</groupId>
+    <artifactId>forecast-api</artifactId>
+    <packaging>war</packaging>
+    <version>${ifs.version}</version>
+    <name>Forecast API Maven Webapp</name>
+    <url>http://maven.apache.org</url>
 
     <properties>
-        <ifs.version>2.0.0</ifs.version>
-        <jersey.version>2.12</jersey.version>
+        <ifs.version>2.0.1</ifs.version>
+        <jersey.version>2.26-b03</jersey.version>
         <metrics.version>3.1.0</metrics.version>
         <mvn.compiler.plugin.version>2.0.2</mvn.compiler.plugin.version>
         <java.version>1.7</java.version>
         <swagger.version>1.3.2</swagger.version>
-        <jackson.version>2.4.1</jackson.version>
+        <jackson.version>2.9.0</jackson.version>
         <servlet.api.version>3.0.1</servlet.api.version>
         <asm.version>2.2.0</asm.version>
-        <catalina.version>7.0.52</catalina.version>
+        <catalina.version>7.0.91</catalina.version>
         <war.plugin.version>2.4</war.plugin.version>
         <surefire.plugin.version>2.16</surefire.plugin.version>
         <failsafe.plugin.version>2.16</failsafe.plugin.version>
     </properties>
 
-    <repositories>
-        <repository>
-            <id>aol-cm-public</id>
-            <name>AOL CM Nexus</name>
-            <url>http://cmmaven.cm.aol.com:8081/nexus/content/groups/public/</url>
-        </repository>
-        <repository>
-            <id>central</id>
-            <name>Maven Repository Central</name>
-            <url>http://repo1.maven.org/maven2</url>
-        </repository>
-        <repository>
-            <id>aol-maven-releases</id>
-            <name>AOL</name>
-            <url>http://cmmaven.cm.aol.com:8081/nexus/content/repositories/aol-maven/</url>
-        </repository>
-    </repositories>
-
-  <dependencies>
-      <dependency>
-          <groupId>com.wordnik</groupId>
-          <artifactId>swagger-jersey2-jaxrs_2.10</artifactId>
-          <version>${swagger.version}</version>
-          <exclusions>
```

Comment:
were those exclusions redundant?

Reply:
Yep. It turns out it was being excluded because it was pulling its own jersey dependency. We don't have this problem with the current versions.